src/CSnakes.SourceGeneration/PythonStaticGenerator.cs

```diff
@@ -122,7 +139,14 @@ from f in functionNames
                         using (GIL.Acquire())
                         {
                             logger?.LogDebug("Importing module {ModuleName}", "{{fileName}}");
-                            module = Import.ImportModule("{{fileName}}");
+                            if (!string.IsNullOrEmpty(encodedSource))
+                            {
+                                module = Import.ImportModule("{{fileName}}", Encoding.UTF8.GetString(Convert.FromBase64String(encodedSource)), "{{fileName}}.py");
+                            }
+                            else
+                            {
+                                module = Import.ImportModule("{{fileName}}");
+                            }
```

Comment:
nit: Consider a conditional expression:

```suggestion
                            this.module = !string.IsNullOrEmpty(source)
                                        ? Import.ImportModule("{{fileName}}", Encoding.UTF8.GetString(Convert.FromBase64String(encodedSource)), "{{fileName}}.py");
                                        : Import.ImportModule("{{fileName}}");
```